packages/lesswrong/lib/collections/multiDocuments/schema.ts

```diff
@@ -22,6 +39,10 @@ const schema: SchemaType<"MultiDocuments"> = {
     optional: true,
     nullable: false,
     canRead: ['guests'],
+    onCreate: async ({ newDocument }) => {
+      const basicSlug = slugify(newDocument.title ?? newDocument.tabTitle);
+      return await getUnusedSlugByCollectionName('MultiDocuments', basicSlug, true);
```

Comment:
`getUnusedSlugByCollectionName` isn't aware of deletions, so if some of the previous-import-clearing in the import script sets the deleted flag without also setting the slug to something out-of-the-way, this will lead to each import having a higher number stuck onto it, and maybe issues with links going to the un-suffixed version. I don't think this needs to change here, will keep an eye on this.

Reply:
ack, added a TODO